chromadb/test/property/test_embeddings.py

```diff
@@ -427,14 +471,20 @@ def update_embeddings(self, record_set: strategies.RecordSet) -> None:
     )
     def upsert_embeddings(self, record_set: strategies.RecordSet) -> None:
         super().upsert_embeddings(record_set)
+
+        normalized_ids = (
```

Comment:
the strategies we use to generate record sets for `update` and `upsert` have `can_ids_be_none` set to `False` - we should raise if we ever do.